### PR TITLE
update llvm-sys to v130

### PR DESCRIPTION
**Pull request by dmitris**
update llvm-sys to v130.x

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aya-rustc-llvm-proxy"
-version = "0.3.0"
+version = "0.3.1"
```
Comment (dave-tucker) [inline Cargo.toml:3]: please don't change this or `cargo-release` will get confused

**Review by dave-tucker — CHANGES_REQUESTED** [review]

**Comment by dave-tucker** [comment]
@dmitris per: https://github.com/aya-rs/bpf-linker/pull/12#pullrequestreview-860950262 this will break support for rustc versions that use LLVM so I'd like to understand what issue you're having before we make a breaking change.

File: Cargo.toml (hunk 1 above)
Comment (dmitris) [reply Cargo.toml]: changed back to "0.3.0"

**Comment by dmitris** [comment]
ok never mind - closing

**Pull request closed without merging**